src/css/blocks/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  align-self: flex-start;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px 0 0 16px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./../../images/star.svg");
+  background-position: center;
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("./../../images/star-active.svg");
+}
+
+.reviews {
+  display: inline-block;
+  font-weight: 400;
+  font-size: 10px;
+  margin-left: 22px;
+}
```

Comment:
We can't set margins for blocks

https://en.bem.info/methodology/css/#external-geometry-and-positioning

Reply:
cleaned up